docs/source/changelog.rst

```diff
@@ -8,10 +8,31 @@ Current version
 Changes
 -------
 
-* Remove support for Python 3.6 and 3.7; add support for up to Python 3.11.
+* Remove support for Python 3.6 and 3.7; add support for up to Python 3.11
+  (`#154 <https://github.com/jensengroup/propka/issues/154>`_ and
+  `#150 <https://github.com/jensengroup/propka/pull/150>`_)
 
-v3.4.0
-======
+* Add context manager for ``open_file_for_reading``
+  (`#133 <https://github.com/jensengroup/propka/pull/133>`_)
+
+Fixes
+-----
+
+* Refactor and fix ``MolecularContainer.get_pi()``
```

Comment:
The precision fix was the most relevant change, so I suggest:
```
* Fix precision of ``MolecularContainer.get_pi()``
```

Reply:
Fixed in 0de6387b6909558e94ed690412d91872dd579f70.